beesdoo_website_portal/__openerp__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018 Rémy Taymans <remytaymans@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'BEES coop Website Portal',
+
```

Comment:
Choose a more generic module name. This module may be installed for other cooperative supermarkets

Reply:
We can speak about that. But the purpose of this module is to restrict the possibility to modify some information of the user. This module is very "stupid", so it's designed to have one module for each client that need to restrict something in the personal user form.

If needed we can create a generic module, if we see that many clients want the same restriction.